source/includes/_05-one-time-sync-ekyc.html.md.erb

```diff
@@ -339,11 +339,23 @@ app.post("/checks", (req, res) => {
     <tr>
       <td><code>check_output</code></td>
       <td>Object structured per <code>check_output</code> below</td>
-      <td>If check did not error</td>
+      <td>If check did not erorr or there is no result</td>
       <td>
         Indicates the result data for this check. See the
         <code>check_output</code> section for the full structure of
-        this check.
+        this check. In the absence of a result, PassFort will try to
+        interprete a result from the check_output.
+      </td>
+    </tr>
+    <tr>
+      <td><code>result</code></td>
+      <td>string</td>
+      <td>If there is no check_output</td>
+      <td>
+        Indicates the result of this check e.g. "1+1", "2+2", "Fail".
+        If this is provided and is not empty, this will be the result
+        of the check in PassFort. The result should only be provided 
+        when the check output cannot.
```

Comment:
`cannot` is a bit too vague still. I would say something like "The `result` field should only be populated when the provider does not give enough information to generate a suitable `check_output`, or if the user has explicitly requested to use the raw "provider result" without PassFort's interpretation.